packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -907,21 +1024,43 @@ describe('ReactErrorBoundaries', () => {
       'ErrorBoundary static getDerivedStateFromError',
       'ErrorBoundary componentWillMount',
       'ErrorBoundary render error',
+      'ErrorBoundary constructor',
```

Comment:
I added these expected log statements to make the tests pass I know, `createRoot(..).render` is more efficient than `ReactDOM.render` so I assumed this is a byproduct of these functional differences but that could be wrong.